gradle.properties

```diff
@@ -1,5 +1,5 @@
 group=org.web3j
-version=4.8.1
+version=4.8.2
 org.gradle.caching=true
 org.gradle.parallel=true
 solidityPluginVersion=0.3.0
```

Comment:
Did you release a new version of the solidity plugin ? if so, won't we bump its version ?